README.md

```diff
@@ -22,7 +32,109 @@ The generated files will be placed in `the-output-directory-sys`. You now have t
 
 You'll now have to write another GIR file (take a look to [gtk/Gir.toml](https://github.com/gtk-rs/gtk/blob/master/Gir.toml) for an example).
 
-*This part needs more information about this toml file*
+### TOML file
+
+At the opposite of the FFI mode, this one only generates the specified objects. You can either add the object's fullname to the `generate` array or add it to the `manual` array (but in this case, you'll only be able to use the `sys` object). Example:
+
+```toml
+generate = ["Gtk.Widget", "Gtk.Window"]
+manual = ["Gtk.Button"]
+```
+
+So in here, both `GtkWidget` and `GtkWindow` will be generated whereas `GtkButton` will just provide the `sys` access and you'll need to write Rust-user level API by hand.
+
```

Comment:
Previous comment applied here: if GtkWidget contains function that used Button, then it will be uncommented and fully generated. For this reason may be better swap button in window, as gtkwidget contains `fn get_parent_window(&self) -> Option<gdk::Window>;` that can be used as example